### PR TITLE
Adding audit and pprofile env vars and .env file example

### PR DESCRIPTION
**Pull request by letzya**
Adding audit and pprofile env vars and .env file example

### CODE REVIEW

File: deployments/tyk/docker-compose.yml [hunk 1]
```diff
@@ -11,10 +11,15 @@ services:
       - ./deployments/tyk/volumes/tyk-dashboard/private-key.pem:/opt/tyk-dashboard/private-key.pem
       - ./deployments/tyk/volumes/tyk-dashboard/catalogue.html:/opt/tyk-dashboard/portal/templates/catalogue.html
       - ./deployments/tyk/volumes/tyk-dashboard/navigation.html:/opt/tyk-dashboard/portal/templates/navigation.html
+      - ./deployments/tyk/volumes/tyk-dashboard/audit.log:/opt/tyk-dashboard/audit.log
     environment:
       - TYK_DB_LICENSEKEY=${DASHBOARD_LICENCE:?Dashboard licence missing from Docker environment file .env. Review The Getting Started steps in README.md.}
       - TYK_LOGLEVEL=${DASHBOARD_LOGLEVEL:-info}
       - TYK_INSTRUMENTATION=${INSTRUMENTATION_ENABLED:-0}
+      - TYK_DB_AUDIT_ENABLED=${DB_AUDIT_ENABLED:?please enable auditing}
```
Comment (davegarvey) [inline deployments/tyk/docker-compose.yml:19]: Why is `TYK_DB_AUDIT_ENABLED` being forced as an env var?

File: deployments/tyk/docker-compose.yml [hunk 2]
```diff
@@ -11,10 +11,15 @@ services:
...
+      - TYK_DB_AUDIT_PATH=${DB_AUDIT_PATH:-"/opt/tyk-dashboard/audit.log"}
+      - TYK_DB_AUDIT_DETAILEDRECORDING=${DB_AUDIT_DETAILEDRECORDING:?error audit}
```
Comment (davegarvey) [inline deployments/tyk/docker-compose.yml:21]: Why is `TYK_DB_AUDIT_DETAILEDRECORDING` being forced as an env var?

**Review by davegarvey — COMMENTED** [review]
@letzya why are 2 of the env vars being forced. I.e. I have to have them set in .env file otherwise I get errors. Can this be approached in a less intrusive way?

File: deployments/tyk/docker-compose.yml (hunk 1 above)
Comment (letzya) [reply deployments/tyk/docker-compose.yml:19]: good catch. I added it as a test. Will fix!

**Comment by letzya** [comment]
@davegarvey please review. Thank you

**Review by davegarvey — APPROVED** [review]

**Pull request merged**